### ISSUE TITLE
Multi Label - Support multi label annotation for Deepedit Apps

### ISSUE THREAD

**Issue by SachidanandAlle** [Project-MONAI/MONAILabel#245]
Support multi label annotation for Deepedit Apps.

**Comment by diazandr3s** [comment]
This issue covers a number of changes:

- [ ] Make the 3DSlicer module adjustable based on a YAML or config file that creates buttons according to the number of labels
- [ ] Customize deepedit transforms for multilabel training
- [ ] Redefine training loop for deepedit method

**Comment by diazandr3s** [comment]
Make the DeepEdit work on multi-label tasks. For that, the following changes should be addressed:

- Upgrade UI interface to support multi-label tasks
- Modify training class/script so it works on multi-label tasks (i.e. dataloader, label rotation during training, etc)
- Modify inference class/script to show all labels or one label at a time.
- Create robust name association between label names and numbers predicted by DeepEdit model
- Create a robust error management warning system when working on multi-label tasks

**Comment by SachidanandAlle** [comment]
you can verify and close this

**Comment by diazandr3s** [comment]
- Created a new App for multilabel segmentation tasks using BTCV dataset
- UI interface to support multi-label tasks is being upgraded.
- All DeepGrow transforms were updated to support multilabel segmentation.
